src/app/panels/tab-proxy.js

```diff
@@ -34,7 +34,7 @@ export class TabProxy {
         return
       }
       this.addTab(file, '', () => {
-        this.fileManager.switchFile(file)
+        this.fileManager.open(file)
         this.event.emit('switchFile', file)
```

Comment:
should we also change the name of event to openFile? 'switchFile' makes no sense anymore

Reply:
Okay, that can be done.

Reply:
@yann300 Liana suggested we also rename `switchFile` command for nightwatch to `openFile` to avoid confusion. What do you think?

Reply:
Only for nightwatch or for all the codebase?  (I mean for the event name too?)
If the changes are well scoped I think that makes sense. If there's a lot of internal rewording changes, I would suggest to split that for another PR.

Reply:
It's not so much a change. It can go in this PR.